backend/core/src/auth/controllers/auth.controller.ts

```diff
@@ -37,18 +42,20 @@ export class AuthController {
   @Post("login")
   @ApiBody({ type: LoginDto })
   @ApiOperation({ summary: "Login", operationId: "login" })
-  login(@Request() req): LoginResponseDto {
-    const accessToken = this.authService.generateAccessToken(req.user)
-    return mapTo(LoginResponseDto, { accessToken })
+  async login(
+    @Request() req,
+    @Response({ passthrough: true }) res: ExpressResponse
+  ): Promise<LoginResponseDto> {
+    return mapTo(LoginResponseDto, await this.authService.tokenGen(res, req.user))
   }
 
   @UseGuards(DefaultAuthGuard)
-  @Post("token")
```

Comment:
By deleting this token endpoint won't we run into the case where during the deployment all calls to the old `token` endpoint will fail. Or in other words when we deploy the backend the /auth/token endpoint will disappear so any calls to this endpoint will fail until the UI is deployed

Reply:
I think that when we deploy this without a front end release things would not work either, since the front end would not have a cookie set to allow it to get through our auth checks

so I think either way we would need to have the front end and backend deployed at the same time for both of these